src/components/AuthForm.js

```diff
@@ -20,10 +20,24 @@ function AuthForm (props) {
       }
     }
 
-    const urlForUsers = endpointApiUrl.concat("/","users")
+    const urlForUsers = endpointApiUrl.concat("/users")
   
     let register = await axios.post(urlForUsers, data)
-    let tokenInfo = sessionStorage.setItem('jwt', register.data.user.token);
+    
+    sessionStorage.setItem('jwt', register.data.user.token);
+  }
+
+  async function submitSignInData() {
```

Comment:
I recommend using a `try...catch` statement for all your Axios request. This can help you handle errors with the requests if it ever happens.

Reply:
Addressed. `try catch` statement has been added to all `axios` requests. Thanks!